MyBooking/Views/BookingView/src/App.js

```diff
@@ -0,0 +1,250 @@
+                price: 100,
+                peopleCount: 1,
+                possibleAnimals: true,
+                key: 'asdc123',
```

Comment:
Is this field somehow related to our tables?

Reply:
`key: 'asdc123`

Reply:
It is a temporary value when we make integration with API it will be changed to some id. It's needed for rendering unique cells in react

Reply:
And no, it is not related to out tables